pkg/option/config.go

```diff
@@ -1746,6 +1754,7 @@ func (c *DaemonConfig) Populate() {
 	c.CTMapEntriesTimeoutSVCAny = viper.GetDuration(CTMapEntriesTimeoutSVCAnyName)
 	c.CTMapEntriesTimeoutSYN = viper.GetDuration(CTMapEntriesTimeoutSYNName)
 	c.CTMapEntriesTimeoutFIN = viper.GetDuration(CTMapEntriesTimeoutFINName)
+	c.PolicyAuditMode = viper.GetBool(PolicyAuditModeArg)
```

Comment:
I think you need the corresponding CLI argument defined in `daemon/daemon_main.go` and to use `option.BindEnv()` to bind it to the corresponding environment variable to allow this to be configured in k8s setups?

Reply:
Thanks for mentioning that. Should be fixed now.